src/main/java/com/harrison/config/RabbitConfig.java

```diff
@@ -0,0 +1,46 @@
+package com.harrison.config;
+
+import org.reflections.vfs.Vfs;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.reflections.vfs.Vfs'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://52.38.14.86:9000/sonar/coding_rules#rule_key=squid%3AUselessImportCheck)